app/javascript/controllers/edit_deposit_controller.js

```diff
@@ -31,6 +31,7 @@ export default class extends Controller {
     const [data, _status, xhr] = event.detail;
     switch (xhr.status) {
       case 400:
+        this.clearErrors()
```

Comment:
So we clear the errors before they're parsed? I'm a little surprised to see us clearing errors when we get a 400 back, but maybe I'm just not following closely enough.

Reply:
from discussing w/ jcoyne and testing locally, it seemed to me that we got all the errors back at once when any server-side errors were transmitted, and so it was safe to clear everything, and then re-flag the things that are still in error based on the `400`.

Reply:
OK.